Fabric.Identity.API/Quickstart/Account/AccountOptions.cs

```diff
@@ -21,7 +21,9 @@ public class AccountOptions
         public static bool WindowsAuthenticationEnabled = true;
         public static bool IncludeWindowsGroups = true;
         // specify the Windows authentication schemes you want to use for authentication
-        public static readonly string[] WindowsAuthenticationSchemes = new string[] { "Negotiate", "NTLM" };
+        // In aspnetcore 2.2 the GetAllSchemesAsync method doesn't return NTLM or Negotiate
+        // for windows authentication, the only one returned now is Windows.
+        public static readonly string WindowsAuthenticationSchemes = "Windows";
```

Comment:
Does "Windows" mean it will accept both "Negotiate" and "NTLM"?

Reply:
I believe so, that is the only option in AspNetCore 2.2